src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -277,55 +278,55 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
-                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.RuleKey" Header="Code" Width="130">
+                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="Hotspot.Issue.RuleKey" Header="Code" Width="130" MinWidth="50">
```

Comment:
I've set reasonable minimum widths for the columns so they are still more or less usable with the default font size.